src/content/docs/browser/new-relic-browser/browser-pro-features/filterable-geography-webpage-metrics-location.mdx

```diff
@@ -11,28 +18,38 @@ import BrowserGeoZoom from 'images/browser-geo-zoom-051822.png'
+import BrowserFilterGeos2 from 'images/browser-filterable-geos2.png'
 
-The map shows your user data by region, and you can visualize your traffic and error hotspots alongside device type information. This level of visual detail is especially useful for professionals in IT and Operations to make business decisions about peering agreements and CDN usage.
+import BrowserFilterGeos3 from 'images/browser-filter-geos-example.png'
+
+
+With New Relic browser monitoring, the **Filterable geography** UI page provides a world view with color-coded performance information about your front-end experience in cities, regions, and countries anywhere around the world. The map shows your user data by region, so you can visualize your traffic and error hotspots alongside device type information. You can also examine critical Core Web Vitals data so that you can prioritize areas around the globe needing attention.
+
+If you work in IT or operations, this level of visual detail is especially useful for making business decisions about peering agreements and CDN usage. For example, if it's unclear which regions are experiencing higher page load times than normal, use our **Filterable geos** UI to help you track down the root of the problem.
 
 For some tips on how to make the most of the available data, see the [filter examples](#filter).
 
 ## View performance data by location [#apdex-by-region]
 
-Filterable geography functionality requires the [Pro or Pro+SPA agent type](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#agent-types).
+Filterable geography functionality requires the [Pro or Pro+SPA agent type](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#agent-types). If you have not installed this agent type, you may be able to use the [legacy **Geography** UI page](#legacy-geo).
 
 <img
   title="Filterable geography UI"
   alt="Screenshot of Filterable geography UI in New Relic"
-  src={screenBrowserFilterGeography}
+  src={BrowserFilterGeos}
 />
 
+<figcaption>
+  **[one.newrelic.com](https://one.newrelic.com) > Browser > (select an app) > Filterable geos:** Get detailed information about network performance and end users' experience anywhere around the world.
+</figcaption>
+
 To view or sort the performance information by location:
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com) > Browser > (select an app) > Filterable geography**.
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > Browser > (select an app) > Filterable geos**.
 2. Use the dropdown to select the type of performance or usage data you want to view, such as page load or unique session count, average time for various processes, largest contentful paint, first input delay, cumulative layout shift, etc.
-3. To drill down to a [specific area](#functions), mouse over or select any area on the geographical map, or select any of the ten worst performing locations from the list to the right of the map.
```

Comment:
I believe this table is all of the reporting countries, so not just a top ten. Also, it is not always the worst performing. It is only the worst performing when high numbers equals bad. For page load count and unique session count, the top of the list is the best performing.

Reply:
@jodinkansagor  Yikes, I will make that correction. Thanks for setting me straight!